src/content/guidelines/accessibility/keyboard.md

```diff
@@ -1,28 +1,21 @@
 ## Keyboard accessibility
-Keyboard accessibility is important for users who rely on or prefer using a keyboard. Encourage accessibility by providing keyboard functionality to all available content. Common keyboard interactions include using the `Tab` key to select different interactive elements on a page and using the `Enter` key or the Spacebar to activate an in-focus element.
+Keyboard accessibility is required and is helpful for users who rely on or prefer using a keyboard. Make sure to provide keyboard functionality to all interactive content. Common keyboard interactions include using the `Tab` key to select different interactive elements on a page and using the `Enter` key or the `Spacebar` to activate an in-focus element.
+
+[IBM Checkpoint 2.1.1 Keyboard](https://www.ibm.com/able/guidelines/ci162/keyboard.html)
 
 ### Focus indicators
-The `tab` key navigates through various interactive elements on a page. A default visual indicator is provided by the web browser in use. The display is an outlined border around the focused element. When an element is in focus, it can be further activated using the keyboard.
+The `tab` key navigates through all interactive elements on a page in the order they are written. A default visual indicator is provided by the web browser in use. The display is an outlined border around the focused element. When an element is in focus, it can be further activated using the keyboard.
```

Comment:
Could `in the order they are written` potentially be `in the order they appear in the HTML document`? Totally understand if it doesn't make sense for casual readers. Only reason it might be relevant to add this is that the source order of an HTML document when it is written could be different than when it is presented to a user due to things like JavaScript